fuzzer/src/cairo_compiled_programs_fuzzer.rs

```diff
@@ -0,0 +1,541 @@
+#![no_main]
+use cairo_vm::cairo_run::{self, CairoRunConfig};
+use cairo_vm::hint_processor::builtin_hint_processor::builtin_hint_processor_definition::BuiltinHintProcessor;
+use libfuzzer_sys::fuzz_target;
+use std::fs;
+use std::process::Command;
+use std::sync::atomic::{AtomicUsize, Ordering};
+
+// Global counter for fuzz iteration
+static FUZZ_ITERATION_COUNT: AtomicUsize = AtomicUsize::new(0);
+
+fuzz_target!(|data: (u128, u8, u8, u128, u128)| {
```

Comment:
If this uses `arbitrary::Arbitrary`, we could use the utility functions we used for `Felt252` testing ([`felt/src/arbitrary_*.rs`](https://github.com/lambdaclass/cairo-vm/blob/main/felt/src/arbitrary_lambdaworks.rs)), or just use `Felt252` directly (it implements `Arbitrary`). See [`felt/src/lib_*.rs`](https://github.com/lambdaclass/cairo-vm/blob/main/felt/src/lib_lambdaworks.rs) for examples.

Reply:
could you check this @dafifynn ?